### PR TITLE
Mooshimity.BrickHill version 0.3.0.0

### PR DESCRIPTION
**Pull request by Markandpreston1231**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/124879)

### CODE REVIEW

**Comment by Markandpreston1231** [comment]
@microsoft-github-policy-service agree

**Comment by Markandpreston1231** [comment]
In Brick Hill's website, clicking on the download button takes you to this url.

File: manifests/m/Mooshimity/BrickHill/0.3.0.0/Mooshimity.BrickHill.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: Mooshimity.BrickHill
+PackageVersion: 0.3.0.0
+PackageLocale: en-US
+Publisher: Mooshimity
+PackageName: Brick Hill
```
Comment (stephengillie) [inline manifests/m/Mooshimity/BrickHill/0.3.0.0/Mooshimity.BrickHill.locale.en-US.yaml:8]:
```suggestion
PublisherUrl: https://www.brick-hill.com/
PrivacyUrl:   https://www.brick-hill.com/privacy
PackageName: Brick Hill
PackageUrl: https://www.brick-hill.com/client/
```

**Comment by stephengillie** [comment]
> In Brick Hill's website, clicking on the download button takes you to this url.

Yes - on the Client page. Our robot had trouble finding this and needed a human to help.

**Comment by Markandpreston1231** [comment]
thx

**Review by stephengillie — APPROVED** [review]

**Pull request merged**